examples/storage/simpletest/hello-pod.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": "hello-persistence",
+  "kind": "Pod",
+  "apiVersion":"v1beta2",
```

Comment:
Don't use v1beta2. Either use v1beta1 or v1beta3. I prefer the latter, since we're in the process of converting all the examples to v1beta3 now.